x-pack/plugins/enterprise_search/server/routes/enterprise_search/engines.test.ts

```diff
@@ -26,22 +26,42 @@ describe('engines routes', () => {
   describe('GET /internal/enterprise_search/engines', () => {
```

Comment:
Do we need a separate PR to rename `/internal/enterprise_search/engines` to something like `/internal/enterprise_search/search_applications`?

Reply:
Currently all the work goes to [engine(s)](https://github.com/saarikabhasi/kibana/tree/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components) folder and there are other instance where term `engines` is being used

Hence, I think it would be easier to combine all of the renaming task to one PR.